### PR TITLE
Add mysql parse to sqlstatement and format sql example

### PR DESCRIPTION
**Pull request by zjcnb**
Add mysql parse to sqlstatement and format sql example

### CODE REVIEW

File: examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/format/MySQLParserFormatExample.java [hunk 1]
```diff
@@ -15,22 +15,37 @@
...
+    
+    private static final String DML_SELECT_SQL = "select age as b, name n from table1 join table2 where id=1 and name='lu';";
+    
+    private static final String DML_INSERT_SQL = "insert into user (name, age, status) values ('z', 18, 1);";
+    
+    private static final String DML_DELETE_SQL = "delete from user where id = 1;";
+    
+    private static final String DML_UPDATE_SQL = "update user set name = 'j' where id = 1;";
+    
+    private static final String DDL_CREATE_SQL = "create table user (id bigint(20) PRIMARY KEY AUTO_INCREMENT, name varchar(20), age int(2), status int(1));";
```
Comment (strongduanmu) [inline examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/format/MySQLParserFormatExample.java:36]: @zjcnb Please keep all mysql keywords in uppercase.

File: examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/statement/MySQLParserStatementExample.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    private static final String DDL_DROP_SQL = "drop table table1, table2";
+    
+    private static final String DDL_ALTER_SQL = "alter table table1 drop age";
+    
+    private static final String DDL_SHOW_SQL = "show columns from table1;";
+    
+    public static void main(String[] args) {
+        SQLParserEngine parserEngine = new SQLParserEngine("MySQL");
+        ParseTree tree = parserEngine.parse(DDL_SHOW_SQL, false);
+        SQLVisitorEngine visitorEngine = new SQLVisitorEngine("MySQL", "STATEMENT", new Properties());
```
Comment (strongduanmu) [inline examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/statement/MySQLParserStatementExample.java:56]: @zjcnb Can we use a loop to cover all defined SQL?

File: examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/statement/MySQLParserStatementExample.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        MySQLStatement sqlStatement = visitorEngine.visit(tree);
+        if (sqlStatement instanceof MySQLInsertStatement) {
+            System.out.println((((MySQLInsertStatement) sqlStatement).getColumns()).stream().findFirst().get().getIdentifier().getValue());
```
Comment (strongduanmu) [inline examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/statement/MySQLParserStatementExample.java:59]: @zjcnb Can we use `System.out.println(sqlStatement)` to replace these judgments?

File: examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/format/MySQLParserFormatExample.java (hunk 1 above)
Comment (zjcnb) [reply examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/format/MySQLParserFormatExample.java:36]: ok.

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**